script/Arcueid/Day 6/06_00_ARC06_1_1.txt

```diff
@@ -103,7 +103,7 @@ Perhaps it'd be a good idea to just respond with an appropriate greeting and the
 [sha:6a69fffcea9af62a92df679db6528d59a073b8c0]{
 -- Page 58, Offset 14061.
 -- 逃げよう、なんて甘い考えは秋葉のひと睨みで霧散した。
-A single glare from Akiha was enough to dash my naive thoughts of escape.
+A single glare from Akiha was enough to dash my naïve thoughts of escape.
```

Comment:
i guess i should go ahead and look for other instances of naive then

Reply:
Yeah, a find and replace for naïve wouldn't be a bad idea

Reply:
ok done